aten/src/ATen/native/native_functions.yaml

```diff
@@ -69,6 +69,10 @@
   variants: method
   supports_named_tensor: True
 
+- func: align_to(Tensor(a) self, DimnameList order, int ellipsis_idx) -> Tensor(a)
```

Comment:
this seems to me like an internal helper function that we shouldn't expose to users

Reply:
We're not exposing this to users. The Python `align_to` call wraps this function.

Reply:
I thought every interface defined in native_functions.yaml becomes visible to users?

Reply:
In Python, torch.Tensor is a subclass of torch._C.TensorBase. The native functions interface generates align_to as a method of torch._C.TensorBase. I added an overwrite for `Tensor.align_to` in the Python definition of the class. Since users only interact with torch.Tensor (and not torch._C.TensorBase), they never see the native functions version of align_to.

Reply:
The above workflow I described is how you can override a method. We can also override torch functions as well (and we do so in cases where it is difficult to implement functionality in C++ for). One of these is [`broadcast_tensors`](https://github.com/pytorch/pytorch/blob/a6bb8b52d4cce3ea356354e88ce4d05659bb21de/torch/functional.py#L26-L52) : users cannot call the native_functions variant of it directly, they always to through the functional.py call.

Reply:
There's another long tail of things you can do in the C++ or codegen side to write functions in native_functions.yaml but have them not be directly exposed to Python. I can explain those in more detail if you're interested.

Reply:
thanks, I think I would like to understand that better at some point. I had another case in a PR where I needed to exclude something from here to avoid exposing it.